forem/forem
# Update table style for member index view

## What type of PR is this? (check all applicable)


## Description

The designs for the member index view call for slightly different styling than our usual `crayons-table` layout. This PR updates the layout accordingly.

## Related Tickets & Documents



- Related Issue https://github.com/forem/forem/issues/16911

## QA Instructions, Screenshots, Recordings

- Make sure your have `member_index_view` feature flag enabled and visit `/admin/users` on a desktop size screen
- Check the table matches [the designs](https://www.figma.com/file/Ejc7VELuN00n1SZrkUTBcQ/Member-List-View?node-id=945%3A6102)
- Check there's no regression on the smaller screen layouts

![new table layout](https://user-images.githubusercontent.com/20773163/162689219-a47c8d9a-f1ff-4b57-9c41-d9789be38c4e.png)


### UI accessibility concerns?

N/A

## Added/updated tests?

- [X] No, and this is why: layout only

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Storybook](https://storybook.forem.com/) (for Crayons components)
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
      or in a [forem.dev](http://forem.dev) post
- [X] This change does not need to be communicated, and this is why not: part of a larger issue behind a feature flag
